test/tap/Makefile

```
@@ -3,10 +3,23 @@
-all: tests tests_with_deps unit_tests
+all: tests tests_with_deps unit_tests pgsql_user_sync_assets
 
 .PHONY: debug
-debug: tests tests_with_deps unit_tests
+debug: tests tests_with_deps unit_tests pgsql_user_sync_assets
+
+# CI test workflows restore only the build outputs below src/ and test/.
+# Stage the canonical user-sync sample below test/ so its TAP tests exercise
+# the same files that are shipped to operators without duplicating them.
+.PHONY: pgsql_user_sync_assets
+pgsql_user_sync_assets:
```

checkmake (0.3.2)

[warning] Target body for "pgsql_user_sync_assets" exceeds allowed length of 5 lines (7).

(maxbodylength)